vitrina/orgs/forms.py

```diff
@@ -381,11 +382,18 @@ class Meta:
             "has_api_access",
             "regenerate_api_key",
             "can_write",
+            "can_make_agreements",
         )
 
     def __init__(self, *args, **kwargs):
+        self.user: User = kwargs.pop("user")
         self.object = kwargs.pop("object", None)
         super().__init__(*args, **kwargs)
+        if self.object_model == Organization:
+            if self.user.viisp_organization == self.object:
+                self.fields["can_make_agreements"].disabled = False
+        else:
```

Comment:
`Explicit is better than implicit` + conditions can be combined
```suggestion
        if self.object_model == Organization and self.user.viisp_organization == self.object:
            self.fields["can_make_agreements"].disabled = False
        elif self.object_model != Organization:
```

Reply:
Now sure how is this better/explicit? Logic is that if object_model is Organization, we hide field completely. Otherwise we show it. 
And if user is not viisp logged in/representative, then we disable it.

Reply:
To answer your question:

> Now sure how is this better/explicit?

Using `elif` instead of `else` explicitly states, when that code is executed instead of leaving it implicit and having to calculate it from the initial `if` statement.

The problem would come when the condition in `if` statement changes, because of some additional business logic and then the `else` would have to be changed to `elif` either way.

The other problem is that you are nesting two conditions, with no reason frankly, they can be combined, `else` can be changed to `elif` and the whole code just flows.

Other than that, this is just personal preference and not a rule per say, so you can choose to ignore it, I just forgot to add an "Optional:" suffix before this.